prometheus/generic/setup-lenovo-flex-exporter/tasks/prereqs.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Ensure epel-release is installed
+  yum:
```

Comment:
Same comment as the one in the iLO PR:
https://github.com/redhat-cop/monitoring/pull/44#pullrequestreview-654767130